### DISCUSSION TITLE
Taking a lot of time to connect.

### DISCUSSION THREAD

**Discussion by Yogesh-Dubey-Ayesavi** [flutterjanus/flutter_janus_client#112] [Q&A] [answered]
Great Library

When I was going through demo's almost every feature I noticed more delays in connection than usual. So I want to ask whether it is due to misconfiguration on the client side or something else.

Regards.

**Comment by shivanshtalwar** [comment, marked as answer]
To me it seems normal you might wana introduce loader whenever new participant joins the room

**Reply by Yogesh-Dubey-Ayesavi** [reply]
Yeah I'm going through it. Still improvements have to be done. still reading docs and finding the compatibility with my requirements. 

Can we talk about this yogesh.dubey.0804@gmail.com
